tests/beaker_tests/cisco_bgp/test_bgp_non_defaults.rb

```diff
@@ -116,6 +113,13 @@
   'timer_bgp_holdtime'                     => '110',
 }
 
+expected_values_diff = {
```

Comment:
How to build different in bgplib for the manifest? It is built as a string and add to file manifest.pp.